contracts/router/interfaces/gmx/IGMXV1VaultUtils.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface IGMXV1VaultUtils {
```

Comment:
I don't think we are or will be using this interface later on. What do you think about removing this, and cleaning up other interfaces as well?

Reply:
Think your call here. Simply pulled them from gmx repo so we can simplify the interfaces if you think it's better

Reply:
I think it's better to clean these up, and have a general convention of having the simplified interfaces for external contracts (which I believe is already enforced in other PRs).